docs/src/documentation/03-components/03-information/notificationbadge/03-accessibility.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Accessibility
+redirect_from:
+  - /components/notificationbadge/accessibility/
+---
+
+## Accessibility
+
+The NotificationBadge component has been designed with accessibility in mind, providing a way to display important notifications with proper screen reader support.
+
+### Accessibility Props
+
+The following props are available to improve the accessibility of your NotificationBadge component:
+
+| Name      | Type         | Description                                                                                          |
+| :-------- | :----------- | :--------------------------------------------------------------------------------------------------- |
+| ariaLabel | `string`     | Provides an accessible name for the notification badge that will be announced by screen readers.     |
+| icon      | `React.Node` | The displayed icon. When used, `ariaLabel` should be provided to ensure screen reader accessibility. |
+
+### Automatic Accessibility Features
+
+The NotificationBadge component automatically sets `role="status"` to announce updates to assistive technologies.
+
+### The `ariaLabel` Prop
+
+The `ariaLabel` prop is crucial for making NotificationBadge components accessible:
+
+- It sets the `aria-label` attribute on the badge, providing context for screen readers
+- When provided, it completely replaces what screen readers would announce about the badge's visual content
+- It's essential when using icons or when the numerical content alone isn't sufficient to convey meaning
+- For number-only badges (like "3"), it adds context about what the number represents (e.g., "3 unread messages")
+
+### Best Practices
+
+#### Screen Reader Support
+
+- Always provide a descriptive `ariaLabel` that clearly explains what the notification represents
+- Ensure all `ariaLabel` text is translated to the user's language
+- Make the label concise but informative, focusing on the count and its meaning
+
+#### Using Icons
+
+- When using the `icon` prop, always provide an `ariaLabel` since the icon will replace any children content
+- Choose icons that visually reinforce the meaning of the notification
+- Remember that icon-only badges without `ariaLabel` will be effectively invisible to screen reader users
```

Comment:
Will they? 🤔

Reply:
Well 😅 Not invisible as the word "image" will be said... But yeah, I will remove this sentence, I think other info is enough.